config/proto/probe.proto

```diff
@@ -31,7 +31,7 @@ option go_package = "github.com/googleinterns/step224-2020/config/proto";
 // All Hermes probes should extend this probe config.
 message HermesProbeDef {
   string probe_name= 1;
-  Targets targets = 2;
+  repeated TargetDefinition targets = 2;
```

Comment:
Can you explain why it's called TargetDefinition instead of just Target?

Reply:
It was originally called `TargetDef` as it defines what a target for Hermes is. This was requested to be expanded to `TargetDefinition`. I would not be against simplifying this to `Target`.

Reply:
If there's no need to distinguish between Target and TargetDefinition, rename this to Target. Makes it cleaner.

Reply:
I have renamed this `Target`.